src/index.ts

```
@@ -103,6 +104,59 @@
+
+            console.log(resolvedExport, path, parsedPackage);
+            if (typeof resolvedExport === "string") {
+              path = resolvedExport.replace(/^\.?\/?/, "/");
+            } else if (Array.isArray(resolvedExport) && resolvedExport.length > 0) {
+              path = resolvedExport[0]!.replace(/^\.?\/?/, "/");
+            }
+            console.log(resolvedExport, path, parsedPackage);
+
+            // parsedPackage.path = path;
+
+            console.log(parsedPackage);
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)